packages/desktop-client/src/components/budget/SidebarCategory.tsx

```diff
@@ -117,11 +117,13 @@ export function SidebarCategory({
         </Popover>
       </View>
       <View style={{ flex: 1 }} />
-      <NotesButton
-        id={category.id}
-        style={dragging && { color: 'currentColor' }}
-        defaultColor={theme.pageTextLight}
-      />
+      <View style={{ flexShrink: 0 }}>
+        <NotesButton
+          id={category.id}
+          style={dragging && { color: 'currentColor' }}
```

Comment:
Would it work if we put the `flexShrink` in `NotesButton`'s style prop?

Reply:
No, this doesn't seem to work because the style property is applied [to the `Button`](https://github.com/actualbudget/actual/blob/f604fdaf3616d056b899ae824f97bdda476d4cee/packages/desktop-client/src/components/NotesButton.tsx#L60) which is nested too deep.